app/routes/dashboard.js

```diff
@@ -10,5 +10,31 @@ module.exports = function(views_dir, passport) {
       else { console.log('Yep!'); }
     });
   });
+
+  router.get('/start', authCtrl, function(req, res, next) {
+  	res.sendFile(path.join(views_dir, 'workspacestart.html'), function (err) {
+  		if (err) { console.log(err); }
+  		else { console.log('Workspacestart!'); }
+  	});
+  });
+
+  router.get('/workspaces', authCtrl, function(req, res, next){
+  	res.sendFile(path.join(views_dir, 'workspaces.html'), function (err) {
+  		if (err) { console.log(err); }
+  		else { console.log('WOOOORKSPAAAAAEEEJS!!'); }
```

Comment:
Could you include the error message in the logging (you can keep "WOOOORKSPAAAAAEEEJS!!" too if you want :wink: )

Same goes for /start and /settings